configsamples/bgppeer_with_bfdprofile.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: metallb.io/v1beta1
```

Comment:
that's not critical but maybe we should place the BGPPeer above the BFDProfile

Reply:
@fedepaol I think you addressed all my comments besides this two here.

Reply:
Ah I forgot. Re: the order, I prefer to keep it like this as the focus is the bfd profile